changelogs/fix-8156-obw-checkmarks-go-missing

```diff
@@ -0,0 +1,6 @@
+Significance: patch
+Type: Fix
+
+Added random IDs to SVG checkmarks in stepper component #8222
```

Comment:
@rjchow Could you also add this line to `packages/components/CHANGELOG.md` under the `Unreleased`?

Reply:
Thanks for catching that!